index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Henry's Portfolio</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="wrapper">
+      <section class="header">
+        <div class="top-section">
+          <nav class="nav-bar">
+            <p class="logo-box">My Portfolio</p>
+            <span><i class="fas fa-bars"></i></span>
+          </nav>
```

Comment:
- The nav tag is there which is a good use of a semantic tag.
- I see that the background was part of the style file that had a wrong path, great job working on this part.
- [ ] [REQUIRED] Please, address the requested changes suggested by the previous reviewer. (e.g. logo has a link ("a" tag), it should have the fonts as defined in the design.

- The link in the logo is a great addition to the user experience of your website, which is popular in most websites.

Reply:
Though we were not asked to use any logo, I have made 'My Portfolio' a link.